### DISCUSSION TITLE
LTSP Image file

### DISCUSSION THREAD

**Discussion by JoeyTCLinux** [ltsp/ltsp#599] [General]
The question is how to update an .img file after adding new users ,without having to recreate it, Or do i have to create and replace the existing image file everytime i add features etc. Keep in mind im booting this .img file off a PC via network card.

Thanks Guys.

**Comment by intelliant01** [comment]
For just "adding new users", execute `ltsp initrd`. See https://ltsp.org/man/ltsp-initrd/ 
For "everytime i add features", see https://ltsp.org/man/ltsp-image/

**Reply by JoeyTCLinux** [reply]
That worked, thank you so much.
